main_app/templates/home.html

```diff
@@ -5,11 +5,11 @@
     <div class="col s6">
         <div class="home-content">
             <span class="app-title">PLANT PARENTHOOD</span>
-            <p>Nurturing your home, one plant a time</p>
-            <a href="{% url 'signup' %}" class="btn green darken-4">Signup</a>
+            <p>Our aim is to inspire and provide you the training and knowledge you need to be a proud plant parent</p>
+            <a href="{% url 'signup' %}" class="btn green darken-4 signup">Signup</a>
```

Comment:
Sign-up should go back to the base template. Not logged-in users can land on more pages than home. The public database and corresponding detail pages for example should be a landing page from search engines.

Removing a CTA from the base template make it that you have to manually implement them on every possible landing page template

Reply:
Signup link is back on the navbar. Navbar is now responsive but pending testing dropdown menu.